package.json

```diff
@@ -3,7 +3,18 @@
     "version": "1.0.0",
     "description": "A customizable panel built for Hyprland.",
     "main": "app.ts",
+    "type": "module",
     "scripts": {
+        "build": "meson setup build --wipe --prefix \"/usr/\" \nmeson install -C build",
+        "build:fast": "sudo node esbuild.config.js",
+        "start": "gjs -m /usr/share/hyprpanel/hyprpanel.js",
+
+        "build:dev": "sudo node esbuild.config.js -dev",
+        "dev": "gjs -m ./dist/share/hyprpanel/hyprpanel.js",
```

Comment:
Running build with sudo is a big nono, there should be no need for this.

You can build and copy it to `/tmp/` and then run it from there